### PR TITLE
feat: setup end to end test framework for playwright

### PR DESCRIPTION
**Pull request by Vikrant2520**
- setup the end to end playwright testing framework to help test `deployed-ui` with `deployed-be` services.
- added e2e login test

### CODE REVIEW

File: e2e/.github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
```
Comment (Rajat-Dabade) [inline e2e/.github/workflows/playwright.yml:4]: We initiate pushes and submit pull requests targeting the `develop` branch. Also, we don't have a `master` branch. Can you please review `master` and add `develop`?

File: .gitignore [hunk 2]
```diff
@@ -37,7 +37,7 @@ frontend/src/constants/env.ts
 **/locust-scripts/__pycache__/
 **/__debug_bin
 
-frontend/.env
+.env
```
Comment (YounixM) [inline .gitignore:40]: We need to ignore frontend/.env file too

**Comment by YounixM** [comment]
Apart from @Rajat-Dabade 's comment, rest LGTM ✅

**Comment by palashgdev** [comment]
> Apart from @Rajat-Dabade 's comment, rest LGTM ✅

Same ✅

**Review by palashgdev — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (Vikrant2520) [reply .gitignore:40]: .env should take care of all the `.env` files be it in any folder

**Review by Rajat-Dabade — APPROVED** [review]
LGTM 🚀

**Review by YounixM — APPROVED** [review]

**Pull request merged**